vmdb/spec/tools/fix_auth/cli_spec.rb

```diff
@@ -33,10 +33,34 @@
 
   it "should parse hardcoded password" do
     options = described_class.new.parse(%w(-P hardcoded))
-      .options.slice(:hostname, :username, :password, :hardcode, :databases)
+              .options.slice(:hostname, :username, :password, :hardcode, :databases)
     expect(options).to eq(
       :username  => "root",
       :databases => %w(vmdb_production),
       :hardcode  => "hardcoded")
   end
+
+  it "defaults to updating the database" do
```

Comment:
For a follup: If these are all testing `parse`, maybe put this in the it or in a `"#parse"` context.  I looked up to see what this was testing and it wasn't clear.